src/source/mod.rs

```diff
@@ -412,6 +412,7 @@ where
     }
 
     /// Changes the play speed of the sound. Does not adjust the samples, only the play speed.
+    /// Creates a Speed struct that handles the speed control
```

Comment:
Could you replace this with the documentation from the Sink? I especially would like the note that the pitch will change. Finally could you note how the pitch will change (so at 0.5 speed is it higher or lower). That is something that might not be clear for readers of the doc.

Reply:
@dvdsk I have made the required changes , I also added some part of the doc to the sink.set_speed function to increase accessibility . Please review and merge